common/djangoapps/track/views/__init__.py

```diff
@@ -103,8 +103,12 @@ def user_track(request):
             'activity_object': page,
             'extra_data': json.dumps(data, default=str),
         }
-        attempt_to_store_lrs_record.apply_async(
-            args=[lrs_data], queue=settings.LRS_QUEUE)
+        if settings.LRS_IMPLEMENTATION_VERSION == 'v1':
```

Comment:
I might be inclined to use a more readable name for the approach (it's kinda a strategy) - i.e. "lrs_call_lambda" vs "write_to_lrs" instead of a v1

Reply:
Cool. Updated this.